packages/web-service/src/pages/common/file-upload/file-upload.js

```diff
@@ -1,29 +1,24 @@
 import Joi from 'joi'
 import fs from 'fs'
-import path from 'path'
-import Boom from '@hapi/boom'
 import handler, { errorShim } from '../../../handlers/page-handler.js'
-import { CHECK_YOUR_ANSWERS, FILE_UPLOAD } from '../../../uris.js'
-import { scanFile } from '../../../services/virus-scan.js'
 import { MAX_FILE_UPLOAD_SIZE_MB, TIMEOUT_MS } from '../../../constants.js'
+import { scanFile } from '../../../services/virus-scan.js'
+import { APPLICATIONS } from '../../../uris.js'
 
-export const setData = async request => {
-  const currentFilePlusDirectory = path.join(process.env.SCANDIR, path.basename(request.payload['scan-file'].path))
-
-  // We need to take a filename like: hello.txt
-  // And rename it to something like: {unixTimestamp}.{originalFileName}

```

Comment:
Not sure where you're dealing with someone uploading a file, but not submitting to s3?

Inside `file-upload.js` or `upload-work-schedule.js`

Reply:
We will need a clean up there. Again one for the backlog, I will ask Vikram if we can have a policy on the file system